src/classes/ACCT_IndividualAccounts_TDTM.cls

```diff
@@ -120,32 +115,25 @@ public class ACCT_IndividualAccounts_TDTM extends TDTM_Runnable {
                 	
                 	// HH Account should be created if:
```

Comment:
Might want to update this comment, its not necessarily the HH Acocunt, right? or is it?